lsp/nls/src/requests/goto.rs

```diff
@@ -8,25 +8,44 @@ use crate::{
     server::Server,
 };
 
-fn get_defs(term: &RichTerm, server: &Server) -> Vec<LocIdent> {
-    match term.as_ref() {
-        Term::Var(id) => {
-            if let Some(loc) = server.analysis.get_def(&(*id).into()).map(|def| def.ident) {
-                vec![loc]
-            } else {
-                vec![]
-            }
+fn get_defs(term: &RichTerm, ident: Option<LocIdent>, server: &Server) -> Option<Vec<LocIdent>> {
```

Comment:
Is there a semantic difference between `None` and `Some(vec![])` here?

Reply:
No, but changing it to `Option<Vec>` allows `?` for early return